examples/cra/.storybook/config.js

```diff
@@ -1,3 +1,4 @@
 import { configure } from '@storybook/react';
+import '../src/App.css';
```

Comment:
btw, is it only me, or is it not possible to include an `.css` file from `examples/cra/.storybook` directory?

Reply:
Hi @kusmierz, do you mean you want to include files from the Storybook config folder into your app? That's not possible, but the other way around (as in this example) is.